content/docs/command-reference/plots/diff.md

```diff
@@ -77,12 +77,13 @@ all the current plots, without comparisons.
 - `-x <field>` - field name from which the X axis data comes from. An
   auto-generated `index` field is used by default. See
   [Custom templates](/doc/command-reference/plots#custom-templates) for more
-  information on this `index` field. Column names or numbers are expected for
-  tabular metrics files.
+  information on this `index` field. Column names or indices (when `--no-header`
+  option is specified) are expected for tabular metrics files.
 
 - `-y <field>` - field name from which the Y axis data comes from. The last
-  field found in the `--targets` is used by default. Column names or numbers are
-  expected for tabular metrics files.
+  field found in the `--targets` is used by default. Column names or indices
+  (when `--no-header` option is specified) are expected for tabular metrics
```

Comment:
The problem is that the `index` field is explained right next to the mention of "column indices" so even with that parenthesis, it's easily confusing. Also, the parenthesis about `--no-header` adds to the confusion as that option is described after, not before this. Not sure what to do about this but I think "column number" is a perfectly fine term.

Reply:
Yeah per https://trends.google.com/trends/explore?geo=US&q=column%20number,column%20index I will say that we should just keep "number" here. Cc @shcheklein in case you want to override my decision here (context: https://github.com/iterative/dvc.org/pull/1382#pullrequestreview-427729775). Thanks

![image](https://user-images.githubusercontent.com/1477535/99594911-74312b00-29b9-11eb-9635-7fee1d329788.png)

Reply:
Context is important here:

https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.index.html

`number` is not used,  I guess, since it's hard to distinguish with data type "number". May be there are some other reason. Bottom line- we need to slice this search results by the audience to make these search results relevant.

Reply:
Remove these changes for now. If they are correct I'll make a new PR for the same.

Reply:
I agree that "index" would better than "number" if we didn't use the term for another concept in the same paragraph @shcheklein. That's always been the reason to use "number" here. The trends I presented are just to show that it's an OK alternative. I'm aware that those trends don't filter by field.

Reply:
@jorgeorpinel yep, got it! I guess I didn't understand all the nuances of the question here.

Reply:
Yep, Sorry! This came out of nowhere haha